lib/Conversion/TorchToMhlo/Basic.cpp

```diff
@@ -769,6 +769,38 @@ LogicalResult ConvertAtenOp<AtenContiguousOp>::matchAndRewrite(
   return success();
 }
 
+// AtenCopyOp
+template <>
+LogicalResult ConvertAtenOp<AtenCopyOp>::matchAndRewrite(
+    AtenCopyOp op, OpAdaptor adaptor,
+    ConversionPatternRewriter &rewriter) const {
+
+  Value self = adaptor.getSelf();
+  Value src = adaptor.getSrc();
+  auto selfType = self.getType().cast<RankedTensorType>();
+  auto srcType = src.getType().cast<RankedTensorType>();
+
+  if (!selfType.hasStaticShape() || !srcType.hasStaticShape()) {
```

Comment:
Can you use dynamic broadcast to support dynamic shape as well?

Reply:
Dynamic broadcast is handled automatically after decomposition